MaxText/input_pipeline/_tfds_data_processing_c4_mlperf.py

```diff
@@ -0,0 +1,286 @@
+
```

Comment:
Gotcha, @gobbleturk is convergence step appropriate metrics for comparison? I expect the packing way in MLPerf is helpful in the number of convergence since there is more meaningful tokens with less paddings.  

If we need to collect all data points for different models, it might be a long period. Not sure if we can go back to it in March after getting best gpt3 training perf.

Reply:
No we don't need to collect all data points for different models, just Matt's convergence sweep.

Reply:
* [Visualization of different data format](https://colab.sandbox.google.com/drive/16Idie6590780l__4txwyLpkdSIIBZEcF#scrollTo=CJyHgzfncFUi)
* [TODO]convergence test data points